tests/cupy_tests/core_tests/test_dlpack.py

```diff
@@ -74,6 +89,79 @@ def test_conversion(self, dtype):
         testing.assert_array_equal(
             orig_array.data.ptr, out_array.data.ptr)
 
+    @pytest.mark.parametrize("kwargs", [
+        {}, {"max_version": None}, {"max_version": (1, 0)},
+        {"max_version": (10, 10)}]
```

Comment:
Q: What happens if we pass, say, `max_version = (0, 8)`?

Reply:
We export the unversioned one.

Reply:
I think I can expand the test in this regard.